### PR TITLE
Clarify that bootstrap settings may not be pacemaker defaults

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Changed settings can be seen in the crmsh log file.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1158562
* jsc#DOCTEAM-143

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP5
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
  - [x] 15 SP1
- SLE-HA 12
  - [x] 12 SP5
  - [ ] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by zzhou1** [comment]
LGTM. Thanks Tahlia to push this toward closure

**Review by zzhou1 — APPROVED** [review]

File: xml/ha_resource_constraints.xml [hunk 1]
```diff
@@ -975,11 +975,12 @@
      <term>Value is <literal>0</literal>:</term>
      <listitem>
       <para>
-       This is the default. The resource is placed optimally in the
+       The resource is placed optimally in the
        system. This may mean that it is moved when a <quote>better</quote>
        or less loaded node becomes available. This option is almost
```
Comment (dariavladykina) [inline xml/ha_resource_constraints.xml:980]:
```suggestion
       or less loaded node becomes available. The option is almost
```

Comment (dariavladykina) [reply xml/ha_resource_constraints.xml:980]: Just to shrink the ranks of "this" in 3 sentences in a row :-)

**Review by dariavladykina — COMMENTED** [review]
Hi, a tiny comment here. Thanks!

**Pull request merged**